cmd/immuadmin/command/database.go

```diff
@@ -317,6 +317,45 @@ func (cl *commandline) database(cmd *cobra.Command) {
 		Args: cobra.ExactArgs(0),
 	}
 
+	truncateCmd := &cobra.Command{
+		Use:               "truncate",
+		Short:             "Truncate database (unrecoverable operation)",
+		Example:           "delete --yes-i-know-what-i-am-doing {database_name}",
```

Comment:
```suggestion
		Example:           "truncate --yes-i-know-what-i-am-doing {database_name}",
```

Reply:
probably the flag `retention-period` should be mandatory

Reply:
fixed it